Makefile

```diff
@@ -57,7 +57,12 @@ prepare-tests-files:
 	wget --progress=dot:giga -N -P data/ https://s3-eu-west-1.amazonaws.com/mitie/total_word_feature_extractor.dat
 
 test: clean
-	py.test tests --cov rasa
```

Comment:
If we changed this back to the way it was, do the tests still continue to pass? I know separating the tests into various invocations of pytest works, but I don't think it's as practical as just doing `pytest tests/`. If running just that command alone causes Travis to fail then I'd say it's still possible we have a problem regarding memory usage (given that running the tests using 6 separate processes works OK).

Reply:
Still on my todo list. Will test it in the afternoon. As soon as I have some results, I'll come back to you.